flink-end-to-end-tests/test-scripts/kafka-common.sh

```diff
@@ -75,7 +75,10 @@ function start_kafka_cluster {
 }
 
 function stop_kafka_cluster {
-  $KAFKA_DIR/bin/kafka-server-stop.sh
+ if ! [[ -z $(./bin/kafka-server-stop) ]]; then
```

Comment:
There is missing ".sh" here, which cause FLINK-15428. I'll fix it.
cc @zentol @GJL @tillrohrmann